### DISCUSSION TITLE
Welcome to vscode-arduino Discussions!

### DISCUSSION THREAD

**Discussion by lersi** [vscode-arduino/vscode-arduino#3] [Announcements]
## 👋 Welcome!
  This is the community edition for the Arduino extension for vscode, the original project was left unmaintained by Microsoft.  
  Please discuss here any thing related to the Arduino extension, even things that relate to Microsoft's edition.

**Comment by KN1HWK** [comment]
This is super exciting! I just noticed Microsoft deprecated their version and I was worried this wouldn't be a thing. I LOVE the Arduino extension for VSCode so this is extremely welcome.
